src/OrleansSQLUtils/Storage/DbConnectionFactory.cs

```diff
@@ -16,6 +16,13 @@ private static CachedFactory GetFactory(string invariantName)
             // Seeks for database provider factory classes from GAC or as indicated by
             // the configuration file, see at <see href="https://msdn.microsoft.com/en-us/library/dd0w4a2z%28v=vs.110%29.aspx">Obtaining a DbProviderFactory</see>.       
 
+            if (invariantName == AdoNetInvariants.InvariantNamePostgreSql)
```

Comment:
If you pull Orleans via nuget, there is an expectation that the dependencies are pulled via nuget as well.  
All Orleans*Utils packages should work the same way.  For example, OrleansZooKeeperUtils has a dependency on "ZooKeeperNetEx".  With .Net Core coming, SQL Server won't be included in the framework and will have to be installed separately.  
It'd be great to have 4 packages:

- OrleansAdoUtils (generic)
- OrleansSqlServerUtils
- OrleansMySqlUtils
- OrleansPostgreSqlUtils

The last three would be very small and use the code from OrleansAdoUtils